scriptmodules/emulators/yabasanshiro.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="yabasanshiro"
+rp_module_desc="SEGA Saturn emulator Yaba Sanshiro"
+rp_module_help="ROM Extensions: .cue .chd\n\nCopy your SEGA Saturn ios images to $romdir/saturn"
+rp_module_licence="GPL2 https://github.com/devmiyax/yabause/blob/master/LICENSE"
+rp_module_repo="git https://github.com/devmiyax/yabause.git pi4"
+rp_module_section="exp"
+rp_module_flags="!64bit"
+
+function depends_yabasanshiro() {
+    local depends=(cmake pkg-config python-pip cmake build-essential protobuf-compiler libprotobuf-dev libsecret-1-dev libssl-dev libsdl2-dev libboost-all-dev )
+    getDepends "${depends[@]}"
+}
+
+function sources_yabasanshiro() {
+    gitPullOrClone "$md_build" https://github.com/devmiyax/yabause.git pi4 bd9f235
```

Comment:
I think it can just be `gitPullOrClone` without any params if you've set up rp_module_repo

Reply:
I removed params.